packages/go/test/fixtures/10-go-mod-newer/go.mod

```diff
@@ -0,0 +1,3 @@
+module go-mod
+
+go 1.19
```

Comment:
The probe is testing for `1.19.6`. Does this mean that we'll need to keep updating the probe as newer versions are released? Or is it possible to pin the version in the `go.mod` here?

Reply:
You can only specify the Go version as a `<major>.<minor>` in the `go.mod`. We resolve the full version number using the `versionMap` in `go-helpers.ts`. Right now, `1.19` maps to `1.19.6`. The only time we need to update this go version is if we update the version `1.19` maps to.